osc-server/src/test/java/org/osc/core/broker/service/tasks/conformance/openstack/securitygroup/UpdateOrDeleteK8sSecurityGroupMetaTaskTestData.java

```diff
@@ -0,0 +1,136 @@
+/*******************************************************************************
+ * Copyright (c) Intel Corporation
+ * Copyright (c) 2017
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *******************************************************************************/
+package org.osc.core.broker.service.tasks.conformance.openstack.securitygroup;
+
+import static org.osc.core.broker.model.entities.virtualization.SecurityGroupMemberType.LABEL;
+
+import org.osc.core.broker.job.TaskGraph;
+import org.osc.core.broker.model.entities.appliance.Appliance;
+import org.osc.core.broker.model.entities.appliance.ApplianceSoftwareVersion;
+import org.osc.core.broker.model.entities.appliance.DistributedAppliance;
+import org.osc.core.broker.model.entities.appliance.VirtualSystem;
+import org.osc.core.broker.model.entities.management.ApplianceManagerConnector;
+import org.osc.core.broker.model.entities.management.Domain;
+import org.osc.core.broker.model.entities.virtualization.SecurityGroup;
+import org.osc.core.broker.model.entities.virtualization.SecurityGroupMember;
+import org.osc.core.broker.model.entities.virtualization.VirtualizationConnector;
+import org.osc.core.broker.model.entities.virtualization.k8s.Label;
+import org.osc.core.common.virtualization.VirtualizationType;
+
+public class UpdateOrDeleteK8sSecurityGroupMetaTaskTestData {
+
+    private static final String LABEL_NAME = "LABEL";
+    private static final String LABEL_VALUE = "LABEL_VALUE";
+
+    private static final String NO_MC_POLICY_MAPPING_SUPPORTED_LABELS_1 = "NO_MC_POLICY_MAPING_SUPPORTED_LABELS_1";
```

Comment:
policy mapping are totally irrelevant for these tests. This should not be needed. Ideally this data would nothing named policy mapping, only things that are relevant to this task: A security group with multiple members, single member and no member that is all.